macleann/enchiridion-client
# Teensy optimizations

# Bug fixes and other optimizations (January 14, 2024)

- Reduced the amount of requests to the /playlists endpoint when visiting `/playlists`
- Fixed a bug that was preventing the `Add Playlist` and `My Playlists` buttons from appearing on initial render

### Bug fixes:

Fixes #123 

## Type of change


# How to Test?

Make sure you've followed all installation and setup instructions in the README.

```
git fetch origin nm-optimize
git checkout nm-optimize
npm start
```

- [ ] Navigate to `Playlists`
- [ ] Authenticated buttons should be visible
- [ ] Navigate to `Playlists` again
- [ ] Authenticated buttons should not be visible

# Checklist:
